sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/BoundStorageHandle.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.state;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.context.Context;
+
+/** A bound handle for recording measurements against a particular set of attributes. */
```

Comment:
At some point we should add our boilerplate with warnings about using `internal` classes to all the public internal classes, but that can wait until things are a bit more settled.

Reply:
Ah, if you point me at the boilerplate, I can put it in here, or look at some kind of spotless hook to auto-add it to internal packages.

Reply:
Here's an example: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/common/src/main/java/io/opentelemetry/sdk/internal/RateLimiter.java#L17-L18

Reply:
Ok, added that to all internal packages.  Did a quick look at spotless plugin. Think we COULD add this as an automatic check, but it's more than a 5 minute task, so deferring.